galite-core/src/main/kotlin/org/kopi/galite/visual/form/VBlock.kt

```diff
@@ -1978,13 +1978,15 @@ abstract class VBlock(var title: String,
     callProtectedTrigger(VConstants.TRG_POSTDEL)
   }
 
+  var idsFieldsName = arrayListOf("ID") // We take the table's ID by default as "ID"
+
```

Comment:
I do not agree with creating a list containing all tables id fields names. Here is an example where this can cause problems : 
- assuming we have a block with multiple tables : table(Clients), table(Invoices)
- ID of table Clients is a field named "client"
- ID of table Invoices is a field names "ID", However the table Invoices also has a field named "client"
=> In this case, idFieldsName = arrayOf("ID", "client"), which are two column names of table voices, so it is not guaranteed that the id of Invoices table should be affected to the correct column.

We need to re evaluate the specification of this PR to : 

1. Make sure the variable idField is used for all the tables defined in a block. If this is not the case and idField is defined for only the first table, there is no need to define an array list 
2. Find a way to associate each table to its appropriate column ID and make sure to affect the right column to its appropriate table